### PR TITLE
CI: Add editorconfig-checker workflow when testing the project

### PR DESCRIPTION
**Pull request by hjpotter92**
**What does this pull request do? Explain your changes. (required)**
Adds a job to check against `editorconfig-checker` for file not following editorconfig guide

**Specific updates (required)**
- Runs editorconfig-checker against the project

**How did you test each of these updates (required)**
locally + this PR

**Does this pull request close any open issues?**
nope

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -140,3 +140,21 @@ jobs:
...
         uses: github/codeql-action/analyze@v2
+
+  editorconfig:
+    name: Run editorconfig checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.5.2
+        with:
+          fetch-depth: 0
```
Comment (pwilczynskiclearcode) [inline .github/workflows/test.yaml]: `fetch-depth: 0` is for fetching entire git history/branches. It's not needed here, right?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -140,3 +140,21 @@ jobs:
 
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
+
+  editorconfig:
```
Comment (pwilczynskiclearcode) [inline .github/workflows/test.yaml:144]: Maybe we can just extend `codeql` with `editorconfig-checker`?

File: docker/Dockerfile.debian [hunk 3]
```diff
@@ -1,48 +1,53 @@
...
+
+COPY --from=builder2	/root/livepeer	/usr/bin/livepeer
 
 COPY docker/start.sh .
-RUN chmod +x start.sh
 
-EXPOSE 7935/tcp
-EXPOSE 8935/tcp
-EXPOSE 1935/tcp
+RUN	chmod +x start.sh
```
Comment (pwilczynskiclearcode) [inline docker/Dockerfile.debian]: `COPY --chmod=777 docker/start.sh .`

**Review by pwilczynskiclearcode — APPROVED** [review]

File: .github/workflows/test.yaml (hunk 2 above)
Comment (hjpotter92) [reply .github/workflows/test.yaml:144]: codeql takes ~7m to run; editorconfig can be run parallel to that job, and finishes in less than a minute!

File: .github/workflows/test.yaml (hunk 1 above)
Comment (hjpotter92) [reply .github/workflows/test.yaml]: yeah, will update

File: docker/Dockerfile.debian (hunk 3 above)
Comment (hjpotter92) [reply docker/Dockerfile.debian]: my original instinct was to just delete the whole `Dockerfile.debian`. not sure if it is useful anyway!

cc: @thomshutt / @leszko is the `Dockerfile.debian` used anywhere? it looks to be something made for local development but any reason that the official `livepeer/go-livepeer` image (and its associated `Dockerfile`) is not used for same?

Comment (leszko) [reply docker/Dockerfile.debian]: I believe it's used only in the `localdocker`: https://github.com/livepeer/go-livepeer/blob/master/Makefile#L71

**Pull request merged**